src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -406,43 +407,45 @@ export const NumeralDate = <DateType extends NumeralDateObject = FullDate>({
                     isEnd={isEnd}
                     hasValidationIcon={hasValidationIcon}
                   >
-                    <Textbox
-                      {...(index === 0 && { id: uniqueId })}
-                      disabled={disabled}
-                      readOnly={readOnly}
-                      placeholder={datePart}
-                      value={dateValue[datePart as keyof NumeralDateObject]}
-                      onChange={(e) =>
-                        handleChange(e, datePart as keyof NumeralDateObject)
-                      }
-                      ref={(element) => {
-                        refs.current[index] = element;
-                        if (!inputRef) {
-                          return;
+                    <FormSpacingProvider>
```

Comment:
This is another place where I'm not sure why we're removing any formSpacing from the children. And as with the other places I've commented on this, I'd rather it was more explicit that the purpose was to set the `marginBottom` to `undefined`.

I'm especially puzzled why a different approach was taken in `DateRange`, with explicit `my={0}` in the components. If we do want to reset this here then we could use the same approach, since we know what the children are.

Finally, if we do need to wrap this is a `FormSpacingProvider`, I would rather not wrap each of the 3 inputs in it individually, but wrap all 3 (by moving the provider outside the `dateFormat.map(...)`.